fuse/readonly/mount_unix.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // Mount mounts ipfs at a given location, and returns a mount.Mount instance.
-func Mount(ipfs *core.IpfsNode, mountpoint string) (mount.Mount, error) {
+func Mount(ipfs *core.IpfsNode, mountpoint string, allow_other bool) (mount.Mount, error) {
```

Comment:
Gotcha. I didn't know we can pull the config out that way, this is a whole lot easier.